sgqlc/codegen/schema.py

```diff
@@ -181,10 +177,7 @@ def write_banner(self, text):
 
     @staticmethod
     def has_iface(ifaces, name):
-        for iface in ifaces:
-            if name == iface['name']:
-                return True
-        return False
+        return any(name == iface['name'] for iface in ifaces)
```

Comment:
this one I'm okay, but given how small is the method, I'd not bother with such change

Reply:
A common pattern is that we need to find if some condition holds `for` one or all of the items in a collection.

Using Python's `any()` built-in functions is a more concise way of doing this than using a `for` loop.

`any()` will return True when at least one of the elements evaluates to True.

Reply:
yeah, I know how `any()`  works :-)